src/main/java/com/avispl/symphony/dal/avdevices/microphone/sennheiser/tcc2/dto/device/identification/IdentificationDTO.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright (c) 2023 AVI-SPL, Inc. All Rights Reserved.
+ */
+
+package com.avispl.symphony.dal.avdevices.microphone.sennheiser.tcc2.dto.device.identification;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * IdentificationDTO class includes the visual value
+ *
+ * @author Kevin / Symphony Dev Team<br>
+ * Created on 3/16/2023
+ * @since 1.0.0
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class IdentificationDTO {
```

Comment:
Is it used to transfer the Json data, if not please remove the @JsonIgnoreProperties(ignoreUnknown = true). Please check another Object also